components/multiselect/multiselect.tsx

```diff
@@ -1,115 +1,146 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import ReactSelect from 'react-select';
 
+import {ActionMeta} from 'react-select/src/types';
+import {getOptionValue} from 'react-select/src/builtins';
+import {StateManager} from 'react-select/src/stateManager';
+
 import {Constants, A11yCustomEventTypes} from 'utils/constants.jsx';
 import SaveButton from 'components/save_button.jsx';
 
-import MultiSelectList from './multiselect_list.jsx';
+import MultiSelectList from './multiselect_list';
+
+export type Value = {
```

Comment:
accordingly to the source code there are other properties that can appear in the Value type, like: `delete_at`, `display_name` or `scheme_id`.

You can see an example of that here: https://github.com/mattermost/mattermost-webapp/pull/3884/files#diff-045fe5a2f0d207289de3cb9ff104aa1cR109

Probably we can add now the fields that we know can appear, and then wait until we migrate other parts of the application to discover missed fields.